core/modules.rs

```diff
@@ -1986,9 +1986,9 @@ import "/a.js";
       43
     }
 
-    let ext = Extension::builder("test_ext")
-      .ops(vec![op_test::decl()])
-      .build();
+    deno_core::ops!(deno_ops, [op_test]);
+
+    let ext = Extension::builder("test_ext").ops(deno_ops()).build();
```

Comment:
Could `deno_core::ops!` macro be called directly inside `.ops()`?

Reply:
I believe so. I was holding off on this for a bit just to get everything into the `extension!` macro first, but there's no reason why it can't all just be in a single macro.